docs/source/migration.rst

```diff
@@ -0,0 +1,658 @@
+.. _quickstart:
+
+Forest 2.0: Migration Guide
+===========================
+
+The goals of this guide are to cover changes to the Forest SDK (containing pyquil 2.0, new Quil, Quil Compiler, and QVM),
+and to go through an example of migrating a VQE program from Forest 1.3 (pyQuil 1.9, Quil 1.0) to be able to run on the
+new Forest SDK.
+
+
+.. note::
+
+    For installation & setup, follow the download instructions in the section :ref:`start` at the top of the page.
+
+What's changed
+~~~~~~~~~~~~~~
+
+With the new Forest SDK, users will be able to run pyQuil programs on a downloadable QVM and Quil Compiler!
+
+In the following section, we'll cover the main changes to pyQuil, Quil, the Quil Compiler, and the QVM.
+
+Overview of Updates to Quil and pyQuil
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The primary differences in the programming language Quil 1.0 (as appearing in pyQuil 1.3) and Quil 2 (as appearing in
+2.0) amount to an enhanced memory model. Whereas the classical memory model in Quil 1.0 amounted to an flat bit array of
+indefinite size, the memory model in  Quil 2 is segmented into typed, sized, named regions.
+
+In terms of compatibility with Quil 1.0, this primarily changes how ``MEASURE`` instructions are formulated, since their
+classical address targets must be modified to fit the new framework. In terms of new functionality, this allows angle
+values to be read in from classical memory.
```

Comment:
something about increased speed

Reply:
out of scope

Reply:
this whole file was just moved, not edited